src/chip.tsx

```diff
@@ -56,9 +58,23 @@ type ClickableChipProps = ExclusifyUnion<HrefChipProps | ToChipProps | OnPressCh
 type ChipProps = ExclusifyUnion<SimpleChipProps | ClosableChipProps | ToggleChipProps | ClickableChipProps>;
 
 const Chip: React.FC<ChipProps> = (props: ChipProps) => {
-    const {Icon, children, id, dataAttributes, active, onClose} = props;
+    const {Icon, children, id, dataAttributes, active, badge, onClose} = props;
     const {texts, isDarkMode} = useTheme();
     const overAlternative = useThemeVariant() === 'alternative';
+    const {isTabletOrSmaller} = useScreenSize();
+
+    const paddingLeft = Icon && isTabletOrSmaller ? 16 : 8 && Icon ? 8 : 20 && isTabletOrSmaller ? 20 : 12;
+    const paddingRight = isTabletOrSmaller ? 20 : 12;
+    const paddingIcon = isTabletOrSmaller ? 16 : 8;
+
+    const badgeSize = Number(badge) > 9 ? 24 : badge === true ? 8 : 18;
```

Comment:
you don't need this

Reply:
I made the changes and if I use the paddingLeft as a substitute for the others, it ends up causing misalignment

<img width="612" alt="Com as mudanças" src="https://github.com/Telefonica/mistica-web/assets/123959276/01d0df85-5d69-48a4-aeb2-1c8fcf216173">

Reply:
I was talking about basdgeSize, not paddingLeft